examples/jenkins/pipeline/README.md

```diff
@@ -65,7 +67,57 @@ jenkins template represented by jenkinstemplate.json by running these commands a
 
         $ oc start-build sample-pipeline
 
-    Jenkins will: create an instance of the sample-pipeline job, launch a slave, trigger a build in openshift, trigger a
-deployment in openshift, and tear the slave down.
+    Jenkins will: create an instance of the sample-pipeline job, launch a slave, trigger a build in openshift, trigger a deployment in openshift, and tear the slave down.
 
     If you monitor the pods in your default project, you will also see the slave pod get created and deleted.
+
+
+## Pipeline of Pipelines Example
+
+The `mapsapp-pipeline.yaml` template contains a pipeline that instantiates other pipelines and runs them.
+It shows how more than one pipeline can be launched in parallel and how a single Jenkins pipeline
+can work with multiple source code repositories.
+
+To run this example:
+
+1. Ensure that you have a running OpenShift environment as described in the basic example
+2. Create a new project for your pipeline:
+   ```
+   oc new-project mapsapp
+   ```
+
+3. To instantiate the pipeline via the Web Console, create the template from this directory first:
```

Comment:
you can just instantiate it in the console via the "upload" path instead of installing/registering it.